modules/nw-ptp-operator-metrics-reference.adoc

```diff
@@ -59,50 +83,32 @@ Possible values are 0 (`PASSIVE`), 1 (`SLAVE`), 2 (`MASTER`), 3 (`FAULTY`), 4 (`
 |`openshift_ptp_max_offset_ns`
 |Returns the maximum offset in nanoseconds between 2 clocks or interfaces.
 For example, between the upstream GNSS clock and the NIC (`ts2phc`), or between the PTP hardware clock (`phc`) and the system clock (`phc2sys`).
-Applicable to T-GM clocks only.
 |`{from="master", iface="ens2fx", node="compute-1.example.com", process="ts2phc"} 1.038099569e+09`
 
+|`openshift_ptp_nmea_status`
+|Returns the current status of the NMEA connection.
+NMEA is the protocol that is used for 1PPS NIC connections.
+Possible values are 0 (`UNAVAILABLE`) and 1 (`AVAILABLE`).
+|`{iface="ens2fx",node="compute-1.example.com",process="ts2phc"} 1`
+
 |`openshift_ptp_offset_ns`
 |Returns the offset in nanoseconds between the DPLL clock or the GNSS clock source and the NIC hardware clock.
-Applicable to T-GM clocks only.
 |`{from="phc", iface="CLOCK_REALTIME", node="compute-1.example.com", process="phc2sys"} -9`
 
-|`openshift_ptp_process_restart_count`
-|Returns a count of the number of times the `ptp4l` process was restarted.
-|`{config="ptp4l.0.config", node="compute-1.example.com",process="phc2sys"} 1`
-
-|`openshift_ptp_process_status`
-|Returns a status code that shows whether the PTP process is running or not.
-|`{config="ptp4l.0.config", node="compute-1.example.com",process="phc2sys"} 1`
-
-|`openshift_ptp_threshold`
-a|Returns values for `HoldOverTimeout`, `MaxOffsetThreshold`, and `MinOffsetThreshold`.
-
-* `holdOverTimeout` is the time value in seconds before the PTP clock event state changes to `FREERUN` when the PTP master clock is disconnected.
-* `maxOffsetThreshold` and `minOffsetThreshold` are offset values in nanoseconds that compare against the values for `CLOCK_REALTIME` (`phc2sys`) or master offset (`ptp4l`) values that you configure in the `PtpConfig` CR for the NIC.
-|`{node="compute-1.example.com", profile="grandmaster", threshold="HoldOverTimeout"} 5`
+|`openshift_ptp_phase_status`
+|Returns the status of the DPLL phase for the NIC.
+Possible values are -1 (`UNKNOWN`), 0 (`INVALID`), 1 (`FREERUN`), 2 (`LOCKED`), 3 (`LOCKED_HO_ACQ`), or 4 (`HOLDOVER`).
+|`{from="dpll",iface="ens2fx",node="compute-1.example.com",process="dpll"} 3`
 
 |`openshift_ptp_pps_status`
 |Returns the current status of the NIC 1PPS connection.
 You use the 1PPS connection to synchronize timing between connected NICs.
 Possible values are 0 (`UNAVAILABLE`) and 1 (`AVAILABLE`).
 |`{from="dpll",iface="ens2fx",node="compute-1.example.com",process="dpll"} 1`
 
-|`openshift_ptp_nmea_status`
-|Returns the current status of the NMEA connection.
-NMEA is the protocol that is used for 1PPS NIC connections.
-Possible values are 0 (`UNAVAILABLE`) and 1 (`AVAILABLE`).
-|`{iface="ens2fx",node="compute-1.example.com",process="ts2phc"} 1`
+|`openshift_ptp_process_status`
+|Returns a status code that shows whether the PTP process is running or not.
```

Comment:
whether the PTP processes are running or not

Reply:
Updated as per the suggestion.